### PR TITLE
Fix pubkey hash + update claims

### PR DESCRIPTION
**Pull request by saleel**
## Motivation

Pubkey hash in const.ts is incorrect/outdatd.

## Change Summary

- Setting pubkeyHash to `18063425702624337643644061197836918910810808173893535653269228433734128853484`
- Add revealed data to PCD claims

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [ ] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [change set](https://github.com/anon-aadhaar/anon-aadhaar/blob/main/CHANGELOG.md)
- [ ] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [ ] PR includes documentation if necessary.

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

### CODE REVIEW

File: packages/core/src/core.ts [hunk 1]
```diff
@@ -96,7 +97,18 @@ async function getVerifyKey() {
   return vk
 }
 
-export async function verify(pcd: AnonAadhaarCore): Promise<boolean> {
+export async function verify(
+  pcd: AnonAadhaarCore,
+  pubkeyHash?: string
```
[the remark this answers is not in the document]
Comment (saleel) [reply packages/core/src/core.ts]: Yes, I think we can do this in another PR. We can update contracts as well to have similar logic.

File: packages/core/src/core.ts [hunk 2]
```diff
@@ -72,10 +76,19 @@ export async function prove(
...
-    ageAbove18: anonAadhaarProof.ageAbove18 === '1',
-    gender: anonAadhaarProof.gender,
-    pincode: anonAadhaarProof.pincode,
-    state: convertRevealBigIntToString(anonAadhaarProof.state),
+    ageAbove18:
+      args.revealAgeAbove18.value === '1'
+        ? anonAadhaarProof.ageAbove18 === '1'
+        : null,
+    gender:
+      convertRevealBigIntToString(anonAadhaarProof.gender) === ''
```
Comment (saleel) [inline packages/core/src/core.ts:84]: `convertRevealBigIntToString(anonAadhaarProof.gender) || null` would do the same right? For others as well.

Comment (Meyanis95) [reply packages/core/src/core.ts:84]: No, as the output will be an empty string but `anonAadhaarProof.gender || null` should work.

Comment (saleel) [reply packages/core/src/core.ts:84]: If left side is empty string `||` will take the value on the right - so I think its the same.

Comment (Meyanis95) [reply packages/core/src/core.ts:84]: Oooh!
Let me try

Comment (Meyanis95) [reply packages/core/src/core.ts]: It works 
Thanks :)

**Review by Meyanis95 — APPROVED** [review]
LGTM

**Pull request merged**